EngineLayer/EngineLayer.csproj

```diff
@@ -20,7 +20,7 @@
     <PackageReference Include="Microsoft.ML" Version="1.3.1" />
     <PackageReference Include="Microsoft.ML.FastTree" Version="1.3.1" />
     <PackageReference Include="Microsoft.NETCore.App" Version="2.2.8" />
-    <PackageReference Include="mzLib" Version="1.0.494" />
+    <PackageReference Include="mzLib" Version="1.0.496" />
```

Comment:
Please update <3

Reply:
Sorry I didn't get it. Could you explain more?